lib/compress/zstd_compress.c

```diff
@@ -3357,7 +3380,8 @@ static size_t ZSTD_initCDict_internal(
               const void* dictBuffer, size_t dictSize,
                     ZSTD_dictLoadMethod_e dictLoadMethod,
                     ZSTD_dictContentType_e dictContentType,
-                    ZSTD_compressionParameters cParams)
+                    ZSTD_compressionParameters cParams,
+                    ZSTD_CCtx_params params)
```

Comment:
Don't pass in both `ZSTD_CCtx_params` and `ZSTD_compressionParameters`.

If we're taking `ZSTD_CCtx_params` here now, the `cParams` should be transported inside of them.

Reply:
We'll need to think carefully about the parameters, and make sure everything is sane. In the past we were in a really bad state with parameter adjustment, so we need to be careful to not regress.